nautobot/docs/development/migration/index.md

```diff
@@ -0,0 +1,16 @@
+# Migrating from 1.x to 2.0
+
+This document provides guidance on migrating code from version 1.x to version 2.0.0. It covers the major changes introduced in the new version and outlines the necessary modifications you need to make in your code to ensure compatibility.
+
+## Table of Contents
+
+- New to Nautobot v2.0.0
+    - [New UI](../core/react-ui.md)
+    - [Enhanced Filter Fields](../../release-notes/version-2.0.md#enhanced-filter-fields-2804)
+- Changes to Nautobot in v2.0.0
+    - [Generic Role Model](../../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#generic-role-model)
+    - [Collapse Site, Region to Location](../../release-notes/version-2.0.md#collapse-region-and-site-models-into-location-2517)
```

Comment:
Possibly instead link to `../../user-guide/administration/upgrading/from-v1/region-and-site-data-migration-guide.md#region-and-site-related-data-model-migration-guide-for-existing-nautobot-app-installations` (though that information is in a weird place at present, it should be moved to a file under the app development docs...)